src/Raven.Server/Documents/Indexes/Persistence/IndexOperationBase.cs

```diff
@@ -190,6 +197,142 @@ private static Analyzer GetAnalyzer(string fieldName, string analyzer, Dictionar
             return analyzerInstance;
         }
 
+        private static CoraxAnalyzer GetCoraxAnalyzer(string fieldName, string analyzer, Dictionary<Type, CoraxAnalyzer> analyzers, bool forQuerying, string databaseName)
```

Comment:
Same here - can we move it to Corax specific class?

Reply:
Moved